doc/backend_vars.asciidoc

```diff
@@ -93,6 +93,7 @@ QEMUTHREADS;integer;0;Number of cpu threads used by VM
 QEMUVGA;see qemu -device ?;cirrus;VGA device to use with VM
 QEMU_COMPRESS_QCOW2;boolean;1;compress qcow2 images intended for upload
 QEMU_HUGE_PAGES_PATH;string;undef;Define a path to use huge pages (e.g. /dev/hugepages/)
+QEMU_MAX_MIGRATION_TIME;integer;240;Maximum time a migration to file may take for example for snapshot creation before being forcefully aborted.
```

Comment:
Maybe it is worth mentioning that this is measured in seconds.

Reply:
absolutely. I forgot to mention the unit. Fixed that and will merge.